docs/Design.md

```diff
@@ -1,182 +1,150 @@
-# The Design of CKB Bitcoin SPV
+# CKB Bitcoin SPV Design
 
-CKB Bitcoin SPV is a library, for doing Bitcoin simplified payment
-verification on CKB.
+CKB Bitcoin SPV is a library designed to facilitate Bitcoin simplified payment verification (SPV) on Nervos CKB.
 
 ## Abstract
 
-This document describes the design and explains the technical details of CKB
-Bitcoin SPV, which allows users to do on-chain verification for bitcoin
-transactions on CKB chain.
+This document provides a comprehensive overview of the design and technical details of the CKB Bitcoin SPV. Through this material, developers will acquire the knowledge needed to perform on-chain verification of Bitcoin transactions on Nervos CKB.
 
 ## Background
 
-For understanding this document, the knowledge of the following concepts are
-**required**.\
-But we won't discuss them here, please learn them on their own documents.
+A foundational knowledge of the following concepts is **required** for a better understanding of this document. 
+
+Below are brief explanations of each term along with links to sources for detailed learning:
 
 ### Simplified Payment Verification (SPV)
 
-[Simple Payment Verification][SPV], usually abbreviated to SPV, is noted in
-Bitcoin whitepaper. It allows a transaction recipient to prove that the
-sender has control of the source funds of the payment they are offering
-without downloading the full Blockchain, by utilizing the properties of
-Merkle proofs.
+[Simple Payment Verification][SPV] (SPV) allows a transaction receiver to confirm that the sender possesses and controls the funds being sent without needing to download the entire blockchain. This is achieved using Merkle proofs.
 
-### Bitcoin Difficulty Adjustments
+Refer to the [Bitcoin whitepaper](https://bitcoin.org/bitcoin.pdf) for more details.
 
-Every 2016 blocks (which should take two weeks if this goal is kept
-perfectly), every Bitcoin client compares the actual time it took to
-generate these blocks with the 2 weeks goal and modifies the target by the
-percentage difference.[^1]
+### Bitcoin Difficulty Adjustment
 
-> [!Note]
-> The difficulty re-target being based on the time taken for the previous
-> 2015 blocks instead of 2016 blocks.[^2]
+Bitcoin adjusts the computational difficulty of mining a block every 2016 blocks, which ideally takes two weeks. Every Bitcoin client
+adjusts the mining difficulty by comparing the actual production time of the last 2016 blocks to the intended 2-week period, and modifies the target based on the percentage difference. [^1]
 
-### Bitcoin Merkle Proof
+> [!Note]
+> “The difficulty re-target [is] based on the time taken for the previous 2015 blocks instead of 2016 blocks." [^2]
```

Comment:
```suggestion
> “The difficulty re-target is based on the time taken for the previous 2015 blocks instead of 2016 blocks." [^2]
```

Reply:
The original text states, "... the difficulty re-target **being** based on the time taken for the previous 2015 blocks instead of 2016 blocks." In the quoted sentence, "The difficulty re-target [is] based on the time taken for the previous 2015 blocks instead of 2016 blocks," square brackets are used around "is" to follow the rule of quotation in professional writing, ensuring grammatical consistency.